gateway/modules/modules.go

```
@@ -81,4 +82,23 @@ func New(nodeID string, managers *managers.Managers, globalMods *global.Global)
 // Delete deletes a project
 func (m *Modules) Delete(projectID string) {
 	// Close all the modules here
+	logrus.Debugln("closing config of db module")
+	if err := m.db.CloseConfig(); err != nil {
+		logrus.Errorf("error closing db module config - %s", err.Error())
```

Use `utils.LogError` and ignore the error